src/Components/Header/Header.scss

```
@@ -0,0 +1,126 @@
+@import "../../Styles/common.scss";
+.header {
+  width: $main-width;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  height: 156px;
+  margin-bottom: 10px;
+  /* header Menu List */
+  .header__menu {
+    display: flex;
+    justify-content: flex-end;
+    padding: 15px 10px;
+    font-size: 12px;
+
+    li a {
+      padding: 0 10px 0 10px;
```

아래와 같이 작성하면 좀 더 간결해지겠네요! :) 
```suggestion
      padding: 0 10px;
```